src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const result = days < 3
+    ? days * 40
+    : days < 7
+      ? (days * 40) - 20
+      : days >= 7
+        ? (days * 40) - 50
+        : 0;
```

Comment:
Nested ternary operators are EVIL. Avoid them.
And try to avoid magic numbers. All numbers that are unclear from the context should be put into constants with a name that is understandable in terms of content

Reply:
why ternary op. is Evil ?

Reply:
one ternary operator is good but when there are a lot it's hard to read and understand what's going on